src/index.js

```diff
@@ -84,9 +84,9 @@ function createLoader() {
 function initializePal(loader) {
   let type;
 
-  const isElectronRenderer = isNodeLike && process.type === 'renderer';
+  var isRenderer = isNodeLike && (process.type === 'renderer' || process.versions['node-webkit']);
```

Comment:
I use const too so no idea how I managed that. 

Both `versions['node-webkit']` and `versions['nw']` work for me. I asked on the nwjs gitter channel and `node-webkit` is more backwards compatible so works with versions < 0.12.3 too.